stripe/stripe.lua

```diff
@@ -0,0 +1,84 @@
+--[[
+
+# [Stripe API](https://stripe.com/docs/api) connector.
+
+### Authorization
+
+Authorization use [API key](https://dashboard.stripe.com/account/apikeys).
```

Comment:
Please use the following link instead
`https://stripe.com/docs/dashboard#api-keys`
That one you used required service login.